README.md

```diff
@@ -1,22 +1,76 @@
-# TITLE 
-[![All Contributors](https://img.shields.io/badge/all_contributors-1-orange.svg?style=flat-square)](#contributors)
+<div align="center">
+<h1>FLINT UI</h1>
+<p>
+UI for FLINT.Cloud project which aims to build a continuous deployment pipeline to offer FLINT as a SaaS overcloud
```

Comment:
Please change to a "A FLINT client, written in Vue, to provide an awesome user interface for configuring simulations using the FLINT.Cloud APIs."